README.md

```diff
@@ -404,7 +404,7 @@ Volume requests embedded in Pod spec are provisioned as ephemeral volumes. The v
 
 |key|meaning|optional|values|
 |---|-------|--------|-------------|
-|`size`|Size of the requested ephemeral volume|No||
+|`size`|Size of the requested ephemeral volume, binary units are support ("1k" == 1024 bytes)|No||
```

Comment:
support => supported

Reply:
Other than this and the log message changes that @okartau pointed out, is this PR conceptually okay?

Reply:
I didn't check fully yet but it sounds like useful change. I will review it now.